sdk/storage/inc/blobs/append_blob_client.hpp

```diff
@@ -12,40 +12,127 @@
 
 namespace Azure { namespace Storage { namespace Blobs {
 
+  /**
+   * @brief The AppendBlobClient allows you to manipulate Azure Storage append blobs.
+   *
+   * An append blob is comprised of blocks and is optimized for append operations. When you modify
+   * an append blob, blocks are added to the end of the blob only, via the AppendBlock operation.
+   * Updating or deleting of existing blocks is not supported. Unlike a block blob, an append blob
+   * does not expose its block IDs.
+   */
   class AppendBlobClient : public BlobClient {
   public:
-    // connection string
+    /**
+     * @brief Initialize a new instance of AppendBlobClient.
+     *
+     * @param connectionString A connection string includes the authentication information required
+     * for your application to access data in an Azure Storage account at runtime.
+     * @param containerName The name of the container containing this blob.
+     * @param blobName The name of this blob.
+     * @param options Optional client options that define the transport pipeline policies for
+     * authentication, retries, etc., that are applied to every request.
+     * @return A new AppendBlobClient instance.
+     */
     static AppendBlobClient CreateFromConnectionString(
         const std::string& connectionString,
         const std::string& containerName,
         const std::string& blobName,
         const AppendBlobClientOptions& options = AppendBlobClientOptions());
 
-    // shared key auth
+    /**
+     * @brief Initialize a new instance of AppendBlobClient.
+     *
+     * @param blobUri A uri
+     * referencing the blob that includes the name of the account, the name of the container, and
```

Comment:
The newline does not look right.

Reply:
Ditto for other doc blocks.

Reply:
see https://github.com/Azure/azure-sdk-for-cpp/pull/213#discussion_r447409172